### PR TITLE
Attempt to fix PHPDBG builds

### PR DESCRIPTION
**Pull request by sanmai**
This PR:

- [x] Attempts to fix PHPDBG builds

Ref: https://github.com/sebastianbergmann/phpunit/pull/4486

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -56,7 +56,13 @@ jobs:
       script:
         - if [[ "$DRIVER" == "xdebug" || "$DRIVER" == "none" ]]; then xdebug_enable; fi
         - make tests/benchmark/MutationGenerator/sources tests/benchmark/Tracing/coverage tests/benchmark/Tracing/sources
-        - if [[ "$DRIVER" != "phpdbg" ]]; then $PHPUNIT_BIN; else phpdbg -qrr $PHPUNIT_BIN; fi
+        - |
+          if [[ "$DRIVER" != "phpdbg" ]]; then
+              $PHPUNIT_BIN;
+          else
+              sed -i s/stream_get_contents/@stream_get_contents/ vendor/phpunit/phpunit/src/Util/PHP/Template/*
```
Comment (maks-rafalko) [inline .travis.yml:63]: What do you think about adding a comment above this line explaining that it related to

> Ref: sebastianbergmann/phpunit#4486

will be easier to understand what is going on here in the future

**Review by maks-rafalko — APPROVED** [review]
:clap:

File: .travis.yml (hunk 1 above)
Comment (sanmai) [inline .travis.yml:63]:
```suggestion
              sed -i s/stream_get_contents/@stream_get_contents/ vendor/phpunit/phpunit/src/Util/PHP/Template/* # Should not be required for PHPUnit 9.4.3+, 8.5.9+
```

Comment (sanmai) [reply .travis.yml]: Sure. How about now?

Comment (maks-rafalko) [reply .travis.yml]: :+1:

**Pull request merged**